easybuild/easyblocks/i/impi.py

```diff
@@ -58,6 +58,12 @@ def extra_options():
         }
         return IntelBase.extra_options(extra_vars)
 
+    def prepare_step(self):
+        if LooseVersion(self.version) >= LooseVersion('2017.2.174'):
+            super(EB_impi, self).prepare_step(requires_runtime_license=False)
```

Comment:
Ok, so SHA256 sums on the 'licenced' downloads and the 'free' downloads are the same. Therefore it is no longer possible to provide your licence to these software packages and this PR can go in. People need to be careful about what they are agreeing to when they download the sources as the installation packages will no longer check your licencing during the install...so it is a case of *Freeloader beware!*